src/Raven.Client/Http/RavenCommand.cs

```diff
@@ -44,7 +44,7 @@ public abstract class RavenCommand<TResult>
         public TimeSpan? Timeout { get; protected set; }
         public bool CanCache { get; protected set; }
         public bool CanCacheAggressively { get; protected set; }
-        public string SelectedNodeTagForRequest { get; protected set; }
+        public string SelectedNodeTag { get; protected set; }
```

Comment:
Do we still need it?

Reply:
yes we use this to explicitly select node to execute the command